source/_docs/guides/circleci.md

```diff
@@ -0,0 +1,235 @@
+- A new GitHub repository and continuous integration implementation on CircleCI.
+
+When code is submitted to GitHub for deployment, CircleCI will create a testing environment and run a suite of tests. If tests pass, the code can be merged into the master branch of the codebase. Before this merge is deployed, CircleCI will repeat the testing process against the production branch.
+
+
+## What You’ll Need
+
+- [Composer](https://getcomposer.org/), [Git](https://git-scm.com/download/) and [Terminus](/docs/terminus) running on your local computer.
+- A GitHub account
+- A [Multidev](/docs/multidev/) enabled site on Pantheon. Signup at [Pantheon for Agencies](https://pantheon.io/agencies/pantheon-for-agencies) (free) to enable Multidev.
+- A free [CircleCI](https://circleci.com) account
+
+
+## Create a New Local Project
+1. From your local terminal, use composer to make a new Drupal 8 project based on the [example repo](https://github.com/pantheon-systems/example-drops-8-composer). Our example project below is called `circle`:
+
+    ```bash
+    composer create-project pantheon-systems/example-drops-8-composer circle
+    ```
+
+2. Initialize a local Git repository for your project:
+
+    ```nohighlight
+    cd circle
+    git init
+    git add --force -A .
+    git commit -m "Initial commit."
+    ```
+
+## Host the Project on GitHub
+1. From your GitHub account page, under the **Repositories** tab, click **New**.
+
+    ![Github Dashboard](/source/docs/assets/images/integrations/github_repos.png)
+
+    Give your new repository a name. We've already imported a README file in the previous steps, so don't check the box to create a new one. Privacy options are up to you.
+
+    ![New empty repository](/source/docs/assets/images/integrations/new_repo.png)
+
+    Click **Create Repository** to complete.
+
+2. From your local terminal, use GitHub's instructions to push your project code to the new remote repository:
+
+    ```nohighlight
+    git remote add origin git@github.com:YOUR-ORG/YOUR-PROJECT
+    git push -u origin master
+    ```
+
+3. Once the push is complete on your local, reload the browser to see your files in the GitHub repository:
+
+    ![Initial commit to new repo](/source/docs/assets/images/integrations/first_commit.png)
+
+4. Create a new [GitHub access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/). Keep the token handy so we can use it on CircleCI later. Be sure to give it the "repo" or "private repo" scope to match the privacy settings for the GitHub repository.
+
+## Run the Project on Pantheon
+Now we will spin up a Drupal 8 site, then overwrite the default installation with code from our GitHub repository.
+
+1. From your Pantheon User or Organization Dashboard, [create a new Drupal 8 site](/docs/create-sites) and click **Visit your Pantheon Dashboard**.
```

Comment:
I'm not sure about this, I made a new site from scratch without associating it with an org. So unless that was done automatically due to my user's relation with an org...?